client/src/app/auth/login.tsx

```diff
@@ -1,7 +1,52 @@
+import { ChangeEvent, FormEvent, useState } from "react";
+import Button from "../components/button";
+import TextField from "../components/text-field";
+
 export default function Login() {
   return (
     <div>
       <h1>Log in</h1>
+      <ExecLogIn />
```

Comment:
Don't need to create another component in the same file. You can move what you wrote in `ExecLogIn` to here.